CRM/ManualDirectDebit/Hook/PostProcess/Membership/DirectDebitMandate.php

```diff
@@ -56,6 +56,11 @@ public function saveMandateData() {
    * Sets direct debit mandate values
    */
   private function setMandateValues() {
+    $submitFiles = $this->form->getVar('_submitFiles');
+    if($this->isFileAttached($submitFiles[CRM_ManualDirectDebit_Common_DirectDebitDataProvider::PREFIX . 'authorisation_file'])){
```

Comment:
Please rewrite this part, to improve readibility:

```php
$submitFiles = $this->form->getVar('_submitFiles');
$authorisationFileName = CRM_ManualDirectDebit_Common_DirectDebitDataProvider::PREFIX . 'authorisation_file';
$authorisationFile = $submitFiles[$authorisationFileName];

if ($this->isFileAttached($authorisationFile)) {
  $this->setMandateFile($authorisationFile);
}
```

Although this might be better:
```php
$authorisationFileName = CRM_ManualDirectDebit_Common_DirectDebitDataProvider::PREFIX . 'authorisation_file';

if ($this->isFileAttached($authorisationFileName)) {
  $authFile = $this->getSubmittedFile($authorisationFileName);
  $this->setMandateFile($authFile);
}
```